jaxopt/_src/zoom_linesearch.py

```diff
@@ -37,13 +37,25 @@
 # pylint: disable=g-bare-generic
 # pylint: disable=invalid-name
 
-# Flags are encoded in base 2 to enable bitwise logic
-_FLAG_NOT_DESCENT_DIRECTION = 1
-_FLAG_CURVATURE_COND_NOT_SATSIFIED = 2
-_FLAG_MAX_ITER_REACHED = 4
-_FLAG_INTERVAL_TOO_SMALL = 8
-_FLAG_NAN_INF_VALUES = 16
-_FLAG_NO_STEPSIZE_FOUND = 32
+# Flags to print errors, used in tests
+_WARNING_PREAMBLE = \
```

Comment:
Minor detail: since we are in `_src/` it's ok if you don't prefix all the constants with a leading `_`.